src/mito2/src/cache/file_cache.rs

```diff
@@ -376,12 +383,51 @@ mod tests {
         ObjectStore::new(builder).unwrap().finish()
     }
 
+    #[tokio::test]
+    async fn test_file_cache_ttl() {
+        let dir = create_temp_dir("");
+        let local_store = new_fs_store(dir.path().to_str().unwrap());
+
+        let cache = FileCache::new(
+            local_store.clone(),
+            ReadableSize::mb(10),
+            Some(Duration::from_secs(1)),
+        );
+        let region_id = RegionId::new(2000, 0);
+        let file_id = FileId::random();
+        let key = IndexKey::new(region_id, file_id, FileType::Parquet);
+        let file_path = cache.cache_file_path(key);
+
+        // Get an empty file.
+        assert!(cache.reader(key).await.is_none());
+
+        // Write a file.
+        local_store
+            .write(&file_path, b"hello".as_slice())
+            .await
+            .unwrap();
+
+        // Add to the cache.
+        cache
+            .put(
+                IndexKey::new(region_id, file_id, FileType::Parquet),
+                IndexValue { file_size: 5 },
+            )
+            .await;
+
+        let exist = cache.reader(key).await;
+        assert!(exist.is_some());
+        tokio::time::sleep(Duration::from_secs(2)).await;
+        let non = cache.reader(key).await;
```

Comment:
We might call [run_pending_tasks()](https://docs.rs/moka/latest/moka/future/struct.Cache.html#method.run_pending_tasks) to ensure the eviction task is done.